pkg/authz/authenticate.go

```diff
@@ -311,3 +315,14 @@ func (g *Gatekeeper) handleLoginHint(r *http.Request, ar *requests.Authorization
 		}
 	}
 }
+
+func (g *Gatekeeper) handleAdditionalScopes(r *http.Request, ar *requests.AuthorizationRequest) {
+	if additionalScopes := r.URL.Query().Get("additional_scopes"); additionalScopes != "" {
+		if err := validate.AdditionalScopes(additionalScopes); err != nil {
+			g.logger.Warn(err.Error())
```

Comment:
@dbranco-sdc , please see other logger message for reference. We would need more things that just the error.